src/ast/values.ts

```diff
@@ -30,7 +30,7 @@ interface RawMemberDescription {
 
 function assembleMemberDescriptions(
 	memberDescriptions: { [key: string]: RawMemberDescription },
-	inheritedDescriptions: MemberDescriptions = null
+	inheritedDescriptions: MemberDescriptions | null = null
```

Comment:
Type change. This function is only used within this file so this should be safe.